### DISCUSSION TITLE
quasar vs raw vuejs

### DISCUSSION THREAD

**Discussion by leostereo** [quasarframework/quasar#10393] [General - Components / Directives / etc]
hi guys , I have finished my first quasar project , so far I created 3 small size projects using vue2 + vuetify and 1 middle size project using quasar.
What I founded in quasar is:
Very easy to implement multiple layouts.
Amazing functional and customizable components (I had no need to create or install any third party library).
Very usefull layout helper.
PWA modules (and others) already builted in.
Usefull cli commands.
And Perhaps more.

So , now ... Im thinking ... Do I need to back to vuejs ?
Is there any reason to use vuejs instead of quasar ? 

Just wanted to share my feeling about that.
Anyway ....thanks for creating quasar.
Leandro.

**Comment by ontwikkelfabriek** [comment]
@leostereo 

Good to hear. Tell everyone around you Quasar is awesome!

> Is there any reason to use vuejs instead of quasar ?

when implementing some small adjustments to existing sites, I could consider  `petite-vue`:
https://github.com/vuejs/petite-vue
